test/integration/multikueue/multikueue_test.go

```diff
@@ -651,7 +761,11 @@ var _ = ginkgo.Describe("Multikueue", func() {
 	})
 
 	ginkgo.It("Should remove the worker's workload and job when managers job is deleted", func() {
+		if managerK8sVersion.LessThan(versionutil.MustParseSemantic("1.30.0")) {
+			ginkgo.Skip("the managers kubernetes version is less then 1.30")
+		}
```

Comment:
Should we leave this case for the k8s version less than 1.30?

Reply:
we could.. , done